README.md

```diff
@@ -26,7 +26,10 @@ If you are interested in contributing to EvalAI-CLI, follow our [contribution gu
     git clone https://github.com/Cloud-CV/evalai-cli.git evalai-cli
     ```
 
-3. Create a virtual environment
+3. Create a virtual environment ([Mac users could easily use Homebrew](https://gist.github.com/apavamontri/4516816))
```

Comment:
Remove this: `([Mac users could easily use Homebrew](https://gist.github.com/apavamontri/4516816))` as we are adding this below